Resources/Private/Language/Backend/PageLayout.xlf

```diff
@@ -1,121 +1,110 @@
-<?xml version="1.0" encoding="utf-8" standalone="yes" ?>
-<xliff version="1.0">
-    <file source-language="en" datatype="plaintext" original="messages" date="2014-07-13T20:55:34Z" product-name="templavoilaplus">
-        <header/>
+<?xml version="1.0" encoding="UTF-8"?>
+<xliff version="1.2" xmlns:t3="http://typo3.org/schemas/xliff" xmlns="urn:oasis:names:tc:xliff:document:1.2">
+    <file source-language="en" original="messages" datatype="plaintext" product-name="templavoilaplus" date="2022-05-03T15:47:31+02:00">
+        <header></header>
         <body>
-            <trans-unit id="mlang_tabs_tab" xml:space="preserve">
-                <source>Page</source>
+            <trans-unit id="error.fce.noMapping" resname="error.fce.noMapping">
```

Comment:
Please do not reorder by id ... they are sorted by usage points yet, which may be changed later on.